pkgs/by-name/st/STC/package.nix

```
@@ -0,0 +1,54 @@
+{ lib, stdenv, fetchFromGitHub, writeText }:
+
+stdenv.mkDerivation rec {
+  pname = "STC";
+  version = "v4.2";
+
+  src = fetchFromGitHub {
+    owner = "stclib";
+    repo = pname;
+    rev = version;
+    hash = "sha256-vbFv1jPVeMQC58fVjoNlbB5w3EM/v4NGKqx0XvaJuns=";
+  };
+
+  phases = [ "unpackPhase" "installPhase" ];
```

Setting phases is deprecated:
- #28910
```suggestion
```

you may want
```suggestion
  dontBuild = true;
```

---

Had to add ` dontFixup = true;` for this to work.